example/pubspec.yaml

```diff
@@ -0,0 +1,9 @@
+name: example
+description: An example project that uses Dart recommended lints.
+version: 0.1.0
+
```

Comment:
Should this specify `publish_to: none` ?

Reply:
Done.